packages/dotcom-ui-header/src/components/drawer/topLevelPartials.tsx

```diff
@@ -32,13 +33,13 @@ const Drawer = (props: THeaderProps) => {
         <DrawerTools {...editions} />
         {!props.userIsSubscribed && subscribeAction && <SubscribeButton {...subscribeAction} />}
         <Search />
-        <nav className="o-header__drawer-menu o-header__drawer-menu--primary o-header__drawer-menu--border">
+        <nav className="o-header__drawer-menu" aria-label="Edition switcher">
```

Comment:
**question**: why have the edition switcher and the sections been split into two `<nav>`'s now?

Reply:
This is to replicate the Markup defined in the Origami template here: https://github.com/Financial-Times/origami/blob/310a61cf30874e153231a14197cf93b79e1d159f/components/o-header/src/tsx/drawer.tsx#L104-L125